in-toto-spec.md

```diff
@@ -736,56 +736,79 @@ The `ARTIFACT_RULE` format is the following:
 ```
 
 The `"pattern"` value is a path-pattern that will be matched against paths
-reported in the link metadata, including bash-style wildcards (e.g.,  "\*"). The
+reported in the link metadata, including bash-style wildcards (e.g.,  `"\*"`). The
```

Comment:
I'm unsure of whether we need this escape with the asterisk (I don't know why it came to be in the first place). I think we should remove it

Reply:
Oh you're right, now that the asterisk is inside backticks the escape is rendered too. The asterisk was escaped before so that markdown does not confuse it with a italic markup. I'll remove the escape.